runtime/jcl/common/vectornatives.cpp

```diff
@@ -70,4 +70,29 @@ Java_jdk_internal_vm_vector_VectorSupport_getMaxLaneCount(JNIEnv *env, jclass cl
 	return laneCount;
 }
 
+#if JAVA_SPEC_VERSION >= 25
+jstring JNICALL
+Java_jdk_internal_vm_vector_VectorSupport_getCPUFeatures(JNIEnv *env, jclass clazz)
+{
+	char buf[1024] = {0};
```

Comment:
Please don't do this: clearing more than the first element is just a waste of time.